app/controllers/mannequins_controller.rb

```diff
@@ -69,6 +69,6 @@ def set_mannequin
 
     # Never trust parameters from the scary internet, only allow the white list through.
     def mannequin_params
-      params.require(:mannequin).permit(:photos ,:age, :location, :category, :description, :height, :waist, :chest, :hips, :hair_color, :eyes, :ethnicity)
+      params.require(:mannequin).permit(:age, :location, :category, :description, :height, :waist, :chest, :hips, :hair_color, :eyes, :ethnicity, photos: [] )
```

Comment:
Please write coments explaining what you are doing, I only understood the array at the end because I have done the nested forms before